observatorium/overlays/moc/smaug/thanos/horizontalpodautoscalers/observatorium-thanos-receive-default.yaml

```diff
@@ -0,0 +1,19 @@
+---
+kind: HorizontalPodAutoscaler
+apiVersion: autoscaling/v2beta2
+metadata:
+  name: observatorium-thanos-receive-default
+spec:
+  scaleTargetRef:
+    kind: StatefulSet
+    name: observatorium-thanos-receive-default
+    apiVersion: apps/v1
+  minReplicas: 5
+  maxReplicas: 25
+  metrics:
```

Comment:
I don't see this in the api spec on smaug, are you sure this is available for our HPA?

Reply:
It is deployed rn, so I am pretty sure this api spec is available